src/App.tsx

```diff
@@ -1,61 +1,143 @@
+import React, { useState } from 'react';
 import './App.scss';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { User } from './components/types/User';
```

Comment:
agree
add types
![image](https://user-images.githubusercontent.com/91411692/235454433-6e75f582-9c18-4cba-a751-dec7cb1774a9.png)

Reply:
import { User } from './components/types/User';

Types already added

Reply:
You don't add file `types` to the PR